Documentation/trying-out-rkt.md

```diff
@@ -5,7 +5,7 @@ For a more in-depth guide of a full end-to-end workflow of building an applicati
 
 ## Using rkt on Linux
 
-rkt consists of a single CLI tool, and is currently supported on amd64 Linux. A modern kernel is required but there should be no other system dependencies. We recommend booting up a fresh virtual machine to test out rkt.
+rkt consists of a single CLI tool, and is currently supported on amd64 Linux. A modern kernel is required but there should be no other system dependencies.
```

Comment:
we should note that it can run on other platforms, but that the primary target platform is amd64